nanoplot/1.41.6/README.md

```diff
@@ -0,0 +1,19 @@
+# NanoPlot container
+
+Main tool : [NanoPlot](https://github.com/wdecoster/NanoPlot)
+
+Additional tools:
+
+- nanomath 1.3.0
+- nanoget 1.19.3
```

Comment:
Note to self. I looked these up and these are the current versions of nanomath and nanoget.